cmake/std/atdm/utils/set_build_options.sh

```diff
@@ -89,6 +89,8 @@ elif [[ $ATDM_CONFIG_BUILD_NAME == *"cuda-9.0"* ]]; then
 elif [[ $ATDM_CONFIG_BUILD_NAME == *"cuda-9.2-gnu-7.2.0"* ]] \
   || [[ $ATDM_CONFIG_BUILD_NAME == *"cuda-9.2_gnu-7.2.0"* ]]; then
   export ATDM_CONFIG_COMPILER=CUDA-9.2_GNU-7.2.0
+elif [[ $ATDM_CONFIG_BUILD_NAME == *"cuda-9.2_gnu-6.3.1_openmpi-2.1.1"* ]]; then
```

Comment:
I think this is too specialized to put into this file.  Can you copy the example in cee-rhel6/ and use a custom_builds.sh script for this?